File: status-layout/StatusLibrary/src/main/java/com/status/layout/OnStatusClickListener.java
package com.status.layout;

import android.support.annotation.NonNull;
import android.view.View;

/**
 * by y on 14/07/2017.
 */

public interface OnStatusClickListener {
    void onNorMalClick(@NonNull View view);

    void onLoadingClick(@NonNull View view);

    void onEmptyClick(@NonNull View view);

    void onSuccessClick(@NonNull View view);

    void onErrorClick(@NonNull View view);
}

File: status-layout/StatusLibrary/src/main/java/com/status/layout/OnStatusClickListener.kt
package com.status.layout

import android.view.View

/**
 * by y on 14/07/2017.
 */

interface OnStatusClickListener {
    fun onNorMalClick(view: View)

    fun onLoadingClick(view: View)

    fun onEmptyClick(view: View)

    fun onSuccessClick(view: View)

    fun onErrorClick(view: View)
}
